packages/core/src/custom-values.ts

```diff
@@ -208,6 +252,8 @@ export function getBoxValue(
         return value;
     } else if (value && customTypes[value.type]) {
         return customTypes[value.type].getValue(path, value, node, customTypes);
+    } else if (value.type === 'string') {
```

Comment:
`string` type value can be overridden by the content of the stylesheet:
```css
@st-import [string] from './custom-vals';
:vars {
  x: string(a, b, c);
}
```

Reply:
https://github.com/wix/stylable/issues/2424